src/settings/SettingsModal.tsx

```diff
@@ -91,7 +100,8 @@ export const SettingsModal = (props) => {
               type="checkbox"
               checked={spatialAudio}
               description="This will make a speaker's audio seem as if it is coming from where their tile is positioned on screen. (Experimental feature: this may impact the stability of audio.)"
-              onChange={(e) => setSpatialAudio(e.target.checked)}
+              // @ts-ignore
+              onChange={(event: Event) => setSpatialAudio(event.target.checked)}
```

Comment:
I think you should be able to find an event type that matches - `React.ChangeEvent<HTMLInputElement>` or something, probably.